docs/gettinginvolved/client-testing.rst

```diff
@@ -0,0 +1,53 @@
+.. _cli-testing:
+
+Client Testing
+==============
+
+There are automated acceptance tests for the Flocker CLI on some platforms.
+See :ref:`client-acceptance-tests`.
+
+Sometimes it is useful to manually test CLIs and their installation instructions on various platforms.
+
+OS X
+----
+
+ClusterHQ has a Mac with the ability to start an OS X Virtual Machine.
+An internal document describing how to use this is available at "Infrastructure > OS X Development Machine".
+
+Linux
+-----
+
+To test on various Linux distributions, it is possible to create either Docker containers or Vagrant virtual machines.
+
+Using Docker
+^^^^^^^^^^^^
+
+To create a Docker container, choose a Docker image from the `Docker Hub <https://registry.hub.docker.com>`_, and start it as below:
+
+.. prompt:: bash $
+
+   docker run -i -t ubuntu /bin/bash
+
+or:
+
+.. prompt:: bash $
+
+   docker run -i -t fedora:20 /bin/bash
```

Comment:
So sad.  Okay.

Can you file an issue for factoring the `20` out of all these different places so we can indicate our supported Fedora version in just one place?